web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -98,9 +98,11 @@ const CreditBatches: React.FC<CreditBatchProps> = ({
 
   useEffect(() => {
     const fetchData = (): void => {
-      getBatchesWithSupply(creditClassId).then(sortableBatches => {
-        setBatches(sortableBatches.data);
-      });
+      getBatchesWithSupply(creditClassId)
+        .then(sortableBatches => {
+          setBatches(sortableBatches.data);
+        })
+        .catch(console.error); // eslint-disable-line no-console
```

Comment:
because the REST methods are throwing errors on failed requests, if we don't handle those errors wherever the fetch methods are called, they can crash the app (like this did)